packages/runtime/src/controls/color.ts

```diff
@@ -15,20 +22,20 @@ type ColorControlConfig = {
 export type ColorControlDefinition<C extends ColorControlConfig = ColorControlConfig> = {
   type: typeof ColorControlType
   config: C
+  version?: 1,
```

Comment:
If we're planning on keeping backwards-compatible versions of the control definition we might want to have different versions for the entire definition.

Reply:
As in, keep them separate. e.g. `ColorControlDefinitionV0` vs. `ColorControlDefinitionV1`?

Reply:
Yup